workflow/python/covid19_scrapers/states/maine.py

```diff
@@ -12,38 +11,26 @@
 
 class Maine(ScraperBase):
     """Maine publishes demographic breakdowns of COVID cases (not yet
-    deaths) in a Google Sheet. We scrape their main reporting page to
-    find the link in case it changes.
+    deaths) in CSV files. These files contain the up to date info only,
+    no historical data is given.
 
+    The homepage URL is:
+    https://www.maine.gov/dhhs/mecdc/infectious-disease/epi/airborne/coronavirus/data.shtml
     """
-    REPORT_URL = 'https://www.maine.gov/dhhs/mecdc/infectious-disease/epi/airborne/coronavirus/data.shtml'
+    CASES_BY_COUNTY_URL = 'https://gateway.maine.gov/dhhs-apps/mecdc_covid/cases_by_county.csv'
+    CASES_BY_RACE_URL = 'https://gateway.maine.gov/dhhs-apps/mecdc_covid/cases_by_race.csv'
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
 
-    @staticmethod
-    def __find_table(tables, title):
-        """Find a table with a child whose text contains title."""
-        title_re = re.compile(title)
-        for table in tables:
-            if table.find(text=title_re):
-                return table
-
     def _scrape(self, **kwargs):
-        # Download the data
-        soup = url_to_soup(self.REPORT_URL)
-
-        # Find the Google sheet
-        url = soup.find('a', string=re.compile('Google Sheet', re.I))['href']
-        url = re.sub(r'(.*)/edit\b.*', r'\1/export?format=xlsx', url)
-        _logger.debug(f'Sheets URL is {url}')
-        counties = pd.read_excel(url, sheet_name='cases_by_county')
+        counties = pd.read_csv(self.CASES_BY_COUNTY_URL)
         total_deaths = counties['DEATHS'].sum()
 
-        table = pd.read_excel(url, sheet_name='cases_by_race', index_col=0)
+        table = pd.read_csv(self.CASES_BY_RACE_URL, index_col=0)
         total_cases = table['CASES'].sum()
         known_cases = table['CASES'].drop('Not disclosed').sum()
-        date = table['DATA_REFRESH_DT'].max().date()
+        date = datetime.datetime.strptime(table['DATA_REFRESH_DT'][0], '%Y-%m-%d').date()
```

Comment:
Can we use a `parse_dates=['DATA_REFRESH_DT']` param to `pd.read_csv` instead of parsing here?

Reply:
ah yeah, will fix that up!